module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,19 @@
+#include "vowel.hpp"
+#include <algorithm>
+
+
+void removeVowels(std::vector<std::string>& someText)
+{
+    const std::string vowels{"AEIOUY"};
```

Comment:
What about small letters and I would consider moving it to the header to have it easier to access. Constexpr seems to be better there since we know value before compilation even start.

Reply:
+1

Reply:
I agree about constexpr. Everyting is converted to upper. There is no need to expose vowels list to users.